starter-code/src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App.js";
+
+ReactDOM.render(
+  <div>
+    <App />
+  </div>,
```

Comment:
You really need a wrapper div only when you have to render several compenents or html tags at once, so the div is not really necessary here, but it is okay to put it :)